Cubical/Categories/Displayed/Slice.agda

```diff
@@ -0,0 +1,52 @@
+{-# OPTIONS --safe #-}
+module Cubical.Categories.Displayed.Slice where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+open import Cubical.Data.Sigma
+
+open import Cubical.Categories.Category
+open import Cubical.Categories.Morphism
+open import Cubical.Categories.Functor
+open import Cubical.Categories.Displayed.Base
+open import Cubical.Categories.Displayed.Preorder
+
+open import Cubical.Tactics.CategorySolver.Reflection
+
+private
+  variable
+    ℓC ℓC' ℓD ℓD' : Level
+
+open Category
+open Categoryᴰ
+open Preorderᴰ
+module _ (C : Category ℓC ℓC') where
+  Slices : Categoryᴰ C (ℓ-max ℓC ℓC') ℓC'
+  Slices .Categoryᴰ.ob[_] c = Σ[ c' ∈ C .ob ] (C [ c' , c ])
+  Slices .Categoryᴰ.Hom[_][_,_] f P Q = Σ[ fᴰ ∈ C [ P .fst , Q .fst ] ]
+    f ∘⟨ C ⟩ P .snd  ≡ Q .snd ∘⟨ C ⟩ fᴰ
```

Comment:
Interesting that the triangle becomes a square, but I can see how this gives us the pullback for free

Reply:
To be clear this doesn't give a pullback for free, it avoids the requirement of a pullback